.taskcluster.yml

```diff
@@ -407,75 +407,43 @@ tasks:
+    workerType: github-worker
     metadata:
-      name: "Build/test 64 bit generic-worker on Debian GNU/Linux 8 (jessie)"
-      description: "This builds the 64 bit linux version of generic-worker"
+      name: "docker-worker tests"
+      description: "This run docker-worker tests"
       owner: "{{ event.head.user.email }}" # the user who sent the pr/push e-mail will be inserted here
       source: "{{ event.head.repo.url }}"  # the repo where the pr came from will be inserted here
     extra:
       github:
         # Events that will trigger this task
         events:
           - pull_request.synchronize
-          - push
-    scopes:
-      - secrets:get:repo:github.com/taskcluster/generic-worker
-      - docker-worker:cache:generic-worker-checkout
     payload:
-      features:
-        taskclusterProxy: true
       maxRunTime: 3600
-      image: golang
+      env:
+        GOPATH: "/home/worker/go"
+      image: "ubuntu:18.10"
+      features:
+        dind: true
       command:
         - /bin/bash
         - -vxec
-        - |
-          export CGO_ENABLED=0
-          go version
-          go env
-          curl -s http://taskcluster/secrets/v1/secret/repo:github.com/taskcluster/generic-worker | sed -n 's/.*"b64_encoded_credentials_script": "\(.*\)".*/\1/p' | base64 -d > ~/env_private.sh
-          source ~/env_private.sh
-          mkdir -p "${GOPATH}/src/github.com/taskcluster"
-          cd "${GOPATH}/src/github.com/taskcluster"
-          if [ ! -d generic-worker/.git ]; then rm -rf generic-worker; git clone '{{ event.head.repo.url }}' 'generic-worker'; fi
-          cd 'generic-worker'
-          git fetch '{{ event.head.repo.url }}' "+{{ event.head.ref }}:refs/heads/X${TASK_ID}"
-          git checkout -f "X${TASK_ID}"
-          git reset --hard '{{ event.head.sha }}'
-          git clean -fdx
-          git checkout -B tmp -t "X${TASK_ID}"
-          go get -v -u github.com/taskcluster/livelog github.com/taskcluster/taskcluster-proxy github.com/gordonklaus/ineffassign
-          cd gw-codegen
-          go get -v -u
-          cd ..
-          go generate
-          go get -ldflags "-X main.revision=$(git rev-parse HEAD)" -v -u -t ./...
-          test "$(git status --porcelain | wc -l)" == 0
-          GORACE=history_size=7 CGO_ENABLED=1 go test -timeout 30m -ldflags "-X github.com/taskcluster/generic-worker.revision=$(git rev-parse HEAD)" -v -race ./...
-          "${GOPATH}/bin/ineffassign" .
-      artifacts:
-        public/build/generic-worker-linux-amd64:
-          path: "/go/bin/generic-worker"
-          expires: "{{ '2 weeks' | $fromNow }}"
-          type: file
-      cache:
-        generic-worker-checkout: /go/src
+        - apt-get update && apt-get install -y git golang && mkdir -p "/home/worker/go/src/github.com/taskcluster" && cd "/home/worker/go/src/github.com/taskcluster" && git clone "{{ event.head.repo.url }}" && cd generic-worker && git checkout "{{ event.head.sha }}" && ./docker-test.sh
```

Comment:
It was easier to read when the commands were separated by new lines.

Reply:
I have changed it, I think this comment no longer holds